### DISCUSSION TITLE
Own S3 storage possible

### DISCUSSION THREAD

**Discussion by one-juru** [chaskiq/chaskiq#464] [Q&A]
Hello,

Is it possible to connect Chaskiq to self-hosted S3 storage? I would really like the data to be stored on my servers. Is there any docker container I could use for, and what ENV variables would I have to set?

I used Caprover for the installation.

Thanks :)

**Comment by michelson** [comment]
Hello, which kind of storage do you want to use? if it's **minio** it should be no so complicated to adapt since you only have to tweak the endpoint on the aws rest client

**Reply by michelson** [reply]
https://docs.min.io/docs/how-to-use-aws-sdk-for-ruby-with-minio-server.html
